src/routes/namespaces/[namespace]/workflows/_workflow-filters.svelte

```diff
@@ -21,32 +20,73 @@
     Canceled: 'Canceled',
     Terminated: 'Terminated',
   };
+
+  let isAdvancedQuery = $page.query.has('query');
+  let workflowIdFilter = '';
+  let workflowTypeFilter = '';
+
+  const handleToggle =
+    (searchType: 'basic' | 'advanced') =>
+    (event: Event): void => {
+      const element = event.target as HTMLAnchorElement;
+      isAdvancedQuery = searchType === 'advanced';
+
+      if (!isAdvancedQuery) $page.query.delete('query');
+
+      goto(element.href);
+    };
 </script>
 
-<div class="grid grid-cols-5 gap-4">
-  <FilterInput
-    parameter="workflow-id"
-    name="Workflow ID"
-    value={workflowIdFilter}
-  />
-  <FilterInput
-    parameter="workflow-type"
-    name="Workflow Type"
-    value={workflowTypeFilter}
-  />
-  <FilterSelect label="Time Range" parameter="time-range" value="24 hours">
-    {#each durations as value}
-      <Option {value}>{value}</Option>
-    {/each}
-  </FilterSelect>
-  <FilterSelect label="Workflow Status" parameter="status" value={null}>
-    {#each Object.entries(statuses) as [label, value]}
-      <Option {value}>{label}</Option>
-    {/each}
-  </FilterSelect>
-  <Select id="filter-by-relative-time" bind:value={timeFormat}>
-    <Option value={'relative'}>Relative</Option>
-    <Option value={'UTC'}>UTC</Option>
-    <Option value={'local'}>Local</Option>
-  </Select>
-</div>
+<section class="flex flex-col gap-2">
```

Comment:
I can see an eventuality when this becomes it's own component but this is good for now.

Reply:
Man my brain is dumb this is already it's own component.